docs/product-assembly_revising-projects/src/main/asciidoc/modify-project-proc.adoc

```diff
@@ -0,0 +1,13 @@
+[id='mod-sample-project']
+= Modifying the Mortgage Process sample project
+
+The *Mortgage Process* sample project consists of predefined data objects, guided decision tables, guided rules, forms, and a business process. Using the sample project provides a quick way to get acclimated with {PRODUCT}. In a real business scenario, you would create all of the assets by providing data that is specific to your business requirements.
+
+.Procedure
+
+Navigate to the Mortgage Process sample project to view the predefined assets.
+
+. Log in to {CENTRAL} and click *Menu* -> *Design* -> *Projects*.
+. Click image:dots.png[] in the upper-right corner of the screen and select *Try Samples*.
+. Select *Mortgage Process* and click *Ok*. The *Assets* view of the project opens.
+. Click on an asset that you want to modify. All of the assets can be edited to refine your desired project requirements.
```

Comment:
Suggestion - is "to meet your project requirements" better instead of "to refine your desired project requirements"

Reply:
Agree and changed :)